MANIFEST.in

```diff
@@ -0,0 +1,11 @@
+include MANIFEST.in
```

Comment:
This file is of the utmost importance. Without it, our `python -m build` command would have no source files available in the sdist to be later built into a wheel